### PR TITLE
Add March 2023 monthly update. 

### PR DESCRIPTION
**Pull request by kgpai**

### CODE REVIEW

**Comment by kgpai** [comment]
Here is source gist : https://gist.github.com/kgpai/00e9dc77ee18caad99a8d442e4cceace

**Review by mbasmanova — APPROVED** [review]
@kgpai Looks good to me. Thank you, Krishna.

@Yuhta @laithsakka Would you take a look as well?

File: velox/docs/monthly-updates/march-2023.rst [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+============
+
+ * Add `printVector` helper function.
+ * Add support for `copy_from` in GenericWriter.
+ * Add preliminary support for OffProcessExpressionEval operator.
+ * Add quota enforcement checks when allocating large chunks of memory.
+ * Deprecate `reallocate` Api from memory allocator.
+ * Optimize null aware anit-join when used with filter.
+ * Remove usage of CppToType.
+
```
Comment (Yuhta) [inline velox/docs/monthly-updates/march-2023.rst:22]: Also can mention that the aggregate function accumulator memory alignment is now fixed

File: velox/docs/monthly-updates/march-2023.rst [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+ * Add :spark:func:`trim`, :spark:func:`ltrim`, :spark:func:`rtrim`  function.
+ * Add :spark:func:`sha2` function.
+
+
+Hive Connector
+==============
+
+ * Add support for multi-level sub field pruning.
+ * Add support for estimating parquet page headers using `ThriftStreamingTransport`.
+ * Fix crash when reading lazy vectors.
```
Comment (Yuhta) [inline velox/docs/monthly-updates/march-2023.rst:44]: We can remove this, as it's just a bug fix and nothing user should note on their side

**Pull request closed without merging**